FanSpeedControl/FanSpeedControl.module.less

```diff
@@ -0,0 +1,46 @@
+// FanSpeedControl.module.less
+//
+
+@import '../styles/mixins.less';
+@import '../styles/skin.less';
+@import '../styles/variables.less';
+
+.fanSpeedControl {
+	display: inline-flex;
+	position: relative;
+
+	.fanSpeedControlSvg {
+		height: 300px;
+		width: 300px;
+	}
+
+	.valueDisplay {
+		align-items: center;
+		display: flex;
+		height: 120px;
+		justify-content: center;
+		left: 50%;
+		position: absolute;
+		top: 50%;
+		transform: translate(-50%,-50%);
+		width: 210px;
+	}
+
+	.applySkins({
+		.segment {
+			fill: @agate-fanspeedcontrol-color;
+		}
+
+		.fanValue {
```

Comment:
It might be better to place `. fanValue` after `. fanIcon` because it is the order in the render function.
```
<Icon className={css.fanIcon} css={css}>{icon}</Icon>
<span className={css.fanValue}>{value}</span>
```